### DISCUSSION TITLE
Push a notification to clients when there is scheduled maintenance

### DISCUSSION THREAD

**Discussion by adryzz** [ppy/osu#26503] [Ideas] [answered]
After reading the recent osu!status tweet, it would be nice to have notifications in the game client for scheduled maintenance to notify users about the online status.

https://twitter.com/osustatus/status/1743123913297846776
![image](https://github.com/ppy/osu/assets/46694241/4d6690a3-c631-4e51-8958-aa6693076cee)

There are already notifications in the client for when the connection is lost, but when the downtime is scheduled it would be nice to let the user know in advance from within the client.

Something like this
![image](https://github.com/ppy/osu/assets/46694241/0e7df45a-66d3-455e-96ae-b960aed4d780)

Or this, with a message that isn't hardcoded.
![image](https://github.com/ppy/osu/assets/46694241/b2777965-66f9-4a12-b6ba-280556badda5)

**Comment by peppy** [comment, marked as answer]
We do this via main menu notices.

Also this happened once in 15 years.
